### PR TITLE
Migrate Kafka console tool examples

### PR DESCRIPTION
**Pull request by anelook**
# What changed, and why it matters

Migrated https://help.aiven.io/en/articles/5344081-kafka-console-tool-examples-for-testing-aiven-for-apache-kafka

### CODE REVIEW

File: docs/products/kafka/howto/connect-with-command-line.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+Connect to Aiven for Apache Kafka® with command line tools
+==========================================================
+
+These examples show how to connect to an Aiven for Apache Kafka® command line clients.
```
[the remark this answers is not in the document]
Comment (anelook) [reply docs/products/kafka/howto/connect-with-command-line.rst]: You're right! I've updated the text

File: docs/products/kafka/howto/connect-with-command-line.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+Pre-requisites
+--------------
+
+Follow `Apache Kafka documentation <https://kafka.apache.org/downloads>`_ to set up ``kafka-console-producer`` and ``kafka-console-consumer``. For ``kafka-avro-console-producer`` follow the instructions in `its GitHub repository <https://github.com/confluentinc/schema-registry>`_.
```
[the remark this answers is not in the document]
Comment (anelook) [reply docs/products/kafka/howto/connect-with-command-line.rst]: Thank you for detailed review! I added changes to kafka-tools-config-file.rst and covered points 1. 2. and 3. .

kafka-avro-console-producer was the tricky one. It is installation of confluent system (and in the GitHub there is section Installation). I tried to use a shortcut not to install everything, but just get the libraries, but it didn't work, so I opted to ask for help one of the solution architects.

File: docs/products/kafka/howto/connect-with-command-line.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+``SCHEMA_REGISTRY_HOST``     Host name for your schema registry
+``SCHEMA_REGISTRY_PORT``     Port number for your schema registry
+``SCHEMA_REGISTRY_USER``     User name for your schema registry
+``SCHEMA_REGISTRY_PWD``      Password for your schema registry
+========================     ===========================================================================================
+
+Produce messages
+-----------------
+
+With ``kafka-console-producer`` you can send multiple messages into your topic.
```
Comment (TibsAtWork) [inline docs/products/kafka/howto/connect-with-command-line.rst:31]: The `--help` for this command says it reads data from stdin to send to Kafka. I think we need to give an example of this after the command.

File: docs/products/kafka/howto/connect-with-command-line.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+.. code::
+
+    kafka-console-producer --broker-list {HOST}:{PORT} \
+      --topic target-topic \
+      --producer.config {CONFIGURATION_PATH}
+
+Produce messages with schema
+----------------------------
+
+With ``kafka-avro-console-producer`` you can include the schema by connecting to your schema registry
```
Comment (TibsAtWork) [inline docs/products/kafka/howto/connect-with-command-line.rst:42]: Again, it's worth saying that this reads from stdin, and maybe giving a very simple example - but we don't really want to show setting up a schema and so on, so it's not worth going into any detail, I think. The documentation for this command line tool appears to be at https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/kafka-commands.html#consume-avro-records and it *might* be worth linking to.

**Review by TibsAtWork — COMMENTED** [review]
Some changes, a couple of comments.
I plan to run the actual commands tomorrow, so can try for some form of "this is what you type at it" text for the producer commands, if you like.

File: docs/products/kafka/howto/connect-with-command-line.rst (hunk 4 above)
Comment (anelook) [reply docs/products/kafka/howto/connect-with-command-line.rst:49]: Good idea. I added the examples and the docs. The example will work with the provided schema through.

**Comment by anelook** [comment]
Thank you Tibs, I added the suggested changes

**Comment by TibsAtWork** [comment]
👍 Thanks @lornajane I was thinking the same myself.

**Review by TibsAtWork — APPROVED** [review]
Approved - all looks good

**Comment by TibsAtWork** [comment]
Just trying to sort out the `make linkcheck` problems before merging.

**Pull request merged**